src/mrack/transformers/aws.py

```diff
@@ -72,6 +72,9 @@ def create_host_requirement(self, host):
             "spot": self._find_value(host, "spot", None, None),
             "delete_volume_on_termination": del_vol,
         }
-        if self.config.get("subnet_id"):
+        if self.config.get("subnet_ids"):
+            req["subnet_ids"] = self.config.get("subnet_ids")
+        elif self.config.get("subnet_id"):
             req["subnet_id"] = self.config.get("subnet_id")
```

Comment:
What do you think about this? 
```suggestion
        req["subnet_ids"] = self.config.get("subnet_ids", [])
        req["subnet_id"] = self.config.get("subnet_id", "")
```
With get and default you do not need if/elif statements and the empty default would help later to check if the value has been really set.

Reply:
Logic extracted to external function